### PR TITLE
fix(bindings): Disable Default Features in superchain dep

### PR DESCRIPTION
**Pull request by refcell**
### Description

Disables default-features typo in the `superchain-registry` dep in `superchain`.

Also adds unused dep warnings to both `superchain-registry` and `superchain-primitives`.

### CODE REVIEW

**Comment by refcell** [comment]
* **#512** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/superchain-registry/512?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @refcell and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by clabby — APPROVED** [review]

File: justfile [hunk 1]
```diff
@@ -1,4 +1,8 @@
 set positional-arguments
+alias t := cargo-tests
```
Comment (vdamle) [inline justfile:2]: Nice idea! Nit - should we alias these as `ct`, `cl`, `cf` to represent rust/cargo shortcuts? And we could have `gl` for go `lint-all`...

**Review by vdamle — APPROVED** [review]
Approved with a minor comment for consideration.

File: justfile (hunk 1 above)
Comment (refcell) [reply justfile:2]: I really like this idea yes

**Review by geoknee — APPROVED** [review]

**Pull request merged**